src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3228,4 +3228,34 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Fix_All_0" xml:space="preserve">
     <value>Fix All: {0}</value>
   </data>
+  <data name="Symbols" xml:space="preserve">
+    <value>Symbols</value>
+  </data>
+  <data name="SemanticSearch" xml:space="preserve">
+    <value>Semantic Search</value>
+  </data>
+  <data name="Query" xml:space="preserve">
+    <value>Query</value>
+  </data>
+  <data name="Semantic_search_only_supported_on_net_core" xml:space="preserve">
+    <value>Semantic search is only supported when code analysis runs in a separate process on the latest .NET (see Tools > Options > Text Editor > C# > Advanced).</value>
```

Comment:
❗ This is extremely problematic. I would not recommend users enable the required feature, which leaves this feature unusable. The value of semantic search does not outweigh the negative performance impact of running Roslyn in a separate process.

Reply:
https://github.com/dotnet/roslyn/issues/72617